tests/Http/Role/CreateRoleRequestHandlerTest.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Http\Role;
+
+use Broadway\CommandHandling\Testing\TraceableCommandBus;
+use Broadway\UuidGenerator\Testing\MockUuidGenerator;
+use CultuurNet\UDB3\Http\Request\Psr7RequestBuilder;
+use CultuurNet\UDB3\Http\Response\AssertJsonResponseTrait;
+use CultuurNet\UDB3\Http\Response\JsonResponse;
+use CultuurNet\UDB3\Model\ValueObject\Identity\UUID;
+use CultuurNet\UDB3\Role\Commands\CreateRole;
+use Fig\Http\Message\StatusCodeInterface;
+use InvalidArgumentException;
+use PHPUnit\Framework\TestCase;
+
+final class CreateRoleRequestHandlerTest extends TestCase
+{
+    use AssertJsonResponseTrait;
+
+    private CreateRoleRequestHandler $handler;
+
+    private TraceableCommandBus $commandBus;
+
+    private string $uuid = '9714108c-dddc-4105-a736-2e32632999f4';
+
+    protected function setUp()
+    {
+        $this->commandBus = new TraceableCommandBus();
+
+        $this->handler = new CreateRoleRequestHandler(
+            $this->commandBus,
+            new MockUuidGenerator($this->uuid)
+        );
+    }
+
+    /**
+     * @test
+     */
+    public function it_throws_invalid_argument_exception_when_an_empty_name_is_given(): void
+    {
+        $request = (new Psr7RequestBuilder())
+            ->withJsonBodyFromArray(['name' => ''])
```

Comment:
Makes me wonder what happens is `name` is completely missing?

Reply:
I tested it before but removed the test, added it again.